src/Illuminate/Foundation/helpers.php

```diff
@@ -554,10 +554,16 @@ function mix($path, $manifestDirectory = '')
         $manifest = $manifests[$manifestPath];
 
         if (! isset($manifest[$path])) {
-            throw new Exception(
-                "Unable to locate Mix file: {$path}. Please check your ".
-                'webpack.mix.js output paths and try again.'
-            );
+            $mixMessage = "Unable to locate Mix file: {$path}. Please check your ".
+                          'webpack.mix.js output paths and try again.';
+
+            if (app()->environment() === 'production') {
+                \Log::info($mixMessage);
```

Comment:
Sorry - I gave you some bad advice.

It should be `app('log')->info($mixMessage);` as we are inside the `helpers.php` file.